scripts/variantstore/README.md

```diff
@@ -0,0 +1,16 @@
+
+
+
+## Cost estimation for BQ queries
+
+### Using the AoU Service account
+To get cost information for the AoU queries, only the AoU Service account has access. 
+After using the json file to auth as that service account, you can query the 
+`region-us`.INFORMATION_SCHEMA.JOBS_BY_USER table to narrow down the query you are interested in.
+
+    gcloud auth activate-service-account --key-file=aou_wgs_vumc_prod.json
```

Comment:
maybe change the json key name here, I'm nervous about having AOU specific details checked into a public repo